templates/ra_rdgw_private_autoscale_elb.element

```diff
@@ -317,7 +331,15 @@
             },
             "Properties" :
             {
-                "ImageId" : { "Fn::GetAtt" : [ "AmiIdLookup", "Id" ] },
+                "ImageId" :
+                {
+                    "Fn::If" :
+                    [
+                        "UseAmiId",
+                            {"Fn::GetAtt": [ "AmiIdLookup", "Id" ]},
+                            {"Ref" : "AmiId"}
+                    ]
+				},
```

Comment:
Changes the tabs to spaces, and line up the closing brace with the opening brace.